### ISSUE TITLE
Make format possible values more consistent

### ISSUE THREAD

**Issue by ratoaq2** [guessit-io/guessit#365]
This is a feature request:

Currently, BDRip and BRRip are detected as `format: BluRay`:
In my opinion, BDRip and BRRip should be detected separately, since both differs a lot from `BluRay` releases in size and quality.

The same applies for DVDRip: A DVDRip is way different from a DVD release (when compared in quality and size).

WEB-DL and WEBRip are already handled as two different things, so I see no reason to not follow the same pattern when it comes to BDRip, BRRip and DVDRip.

```
    rebulk.regex("Blu-?ray", "BD[59]", "BD25", "BD50", value="BluRay")
    rebulk.regex('BD(?!\d)', 'BD-?Rip', value='BDRip')
    rebulk.regex('BR-?Rip', value='BRRip')
    rebulk.regex("DVD", "VIDEO-?TS", "DVD-?R(?:$|(?!E))",  # "DVD-?R(?:$|^E)" => DVD-Real ...
                 "DVD-?9", "DVD-?5", value="DVD")
    rebulk.regex('DVD-?Rip', value='DVDRip')
```

**Comment by Toilal** [comment]
You are perfectly right ! So we could change this as soon as possible, but I don't wan't to multiply possible values in `format` property. Also, I don't want this change to break tools relying on actual values.

What about keeping `format` property values unchanged, and add a new `source` property with values like `BluRay`, `DVD`, `Web`, etc ..., and bring a `RIP` part in `other` property ? The `format` property could be marked as deprecated and removed in next major release (guessit 3).

It you agree, I'm still open for pull request, you've made really good job with other pull requests :) Thanks.

**Comment by ratoaq2** [comment]
It's definitely one way to go. 
I was more concerned with the other issue `streaming_service` getting too big, not this one. 

If you decide going with `Rip` in the `other` field, we'll not be able to distinguish BDRip and BRRip. I don't know if that is an issue or not. It's really hard to draw a line where we should split them or not. 

To be honest, I don't have a strong opinion on this.  🤔

**Comment by Toilal** [comment]
@ratoaq2 Please use the develop branch now. master branch will now contain last released version. (git flow)

**Comment by ratoaq2** [comment]
Before starting something related to this issue, I was doing some analysis.
I was trying to compare existing behavior with ideal behavior (ofc, from my point of view)

I came up with the following:
- DVB could be more generic. Rename it to PDTV (Pure Digital Television)
- BluRay should be split into BluRay, BDRip, BRRip
- DVD should be split into DVD, DVDRip

The `source` idea having `Rip` in `other` part would work for all cases, except `BDRip` and `BRRip` which we won't be able to differentiate.

A simple solution is to have another value in `other` property:
- Bluray = `source: BluRay`
- BRRip = `source: BluRay` + `other: Rip`
- BDRip = `source: BluRay` + `other: [Rip, Retail]` (Since BDRip source is a Retail bluray disc)

Later this week I'll create a PR with a proposal for this one.

**Comment by ratoaq2** [comment]
Ok, with the given definition:
BDrip: stands for BluRay Disc Rip - ripped directly from a Blu-ray disc
BRRip: is encoded from a Blu-ray release, which is in itself a ripped version 

One way to go is:
- BDRip: `source: BluRay Disc` + `other: Rip`
- BRRip: `source: BluRay Release`

**Comment by ratoaq2** [comment]
Fixed by #452
